Resources/views/CRUD/action_buttons.html.twig

```diff
@@ -9,7 +9,7 @@ file that was distributed with this source code.
 
 #}
 {% spaceless %}
-    {% for item in admin.getActionButtons(action, object|default(null)) %}
+    {% for item in admin.getActionButtons(action, (object is defined and object is not null)?object:null ) %}
```

Comment:
well, yes, it looks that code is the same, but the reality is if i have child object and building the action_buttons then with `object|default(null)` i always getting null in the `getActionButtons` function. It work correctly if the object is a parent or have no parent.

But `(object is defined and object is not null)?object:null ` make it work correctly for both parent and child objects.


now digging inside the twig. Note, my object's entity __toString returns empty string (this is correct, this is how i need that)
```
function _twig_default_filter($value, $default = '')
{
    if (twig_test_empty($value)) {
        return $default;
    }

    return $value;
}
...
function twig_test_empty($value)
{
    if ($value instanceof Countable) {
        return 0 == count($value);
    }

    if (is_object($value) && method_exists($value, '__toString')) {
        return '' === (string) $value;
    }

    return '' === $value || false === $value || null === $value || array() === $value;
}
```

the point here is when this happens`object|default(null)` the `twig_test_empty` returns `true`, and if that is `true` the `_twig_default_filter` returns `default` value which is `null`. That's why even i have correct object class i still receiving the `null` inside the `getActionButtons` which later return only button for the `list` action